src/TenancyServiceProvider.php

```diff
@@ -5,6 +5,7 @@
 namespace Stancl\Tenancy;
 
 use Illuminate\Cache\CacheManager;
+use Illuminate\Foundation\Vite as BaseVite;
```

Comment:
Is this directly part of Laravel without any other dependencies?

Reply:
Yes, directly from Laravel without extra dependencies. Only version number should be [Laravel v9.29.0](https://github.com/laravel/framework/releases/tag/v9.29.0) and above for this to work